tests/unit/test-util/test.adb.js

```diff
@@ -1310,4 +1311,32 @@ describe('utils/adb', () => {
     });
   });
 
+  describe('test exposed adb API functions', () => {
```

Comment:
Nit, `test exposed ...` -> `exports exposed in util.adb` (or something like that )